backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/RunVmCommand.java

```diff
@@ -385,6 +386,19 @@ private void releaseHostDevicesLock() {
         }
     }
 
+    private void warnIfRequiredHostDeviceUsed() {
+        if (!needsHostDevices) {
+            return;
+        }
+        Guid hostId = getVm().getDedicatedVmForVdsList().get(0);
+        String deviceName = hostDeviceManager.findUsedPassthroughHostDevice(getVmId(), hostId);
+        if (deviceName != null) {
+            AuditLogableBase event = Injector.injectMembers(new AuditLogableBase(hostId));
+            event.addCustomValue("HostDeviceName", deviceName);
+            auditLogDirector.log(event, AuditLogType.VM_HOST_DEVICE_USED);
```

Comment:
would an audit log be enough? I'm afraid it's too easy to miss that (the VM started so at best the user will see this (long) after starting the vm, at the worse case the user won't notice that at all) - why not to fail the validation?

Reply:
@nyoxi What do you think? As I remember, we talked about this and decided not to fail the validation.

Reply:
We definitely should not fail to start the VM. But there are two cases that we need to distinguish:

1) when starting VM from UI we should show a visible warning
2) when starting VM from API we don't have a lot of options so a warning in event log will have to do